### PR TITLE
Fix package.xml role error

### PR DESCRIPTION
**Pull request by heyanlong**

### CODE REVIEW

**Comment by jmjoy** [comment]
The link of doc file should be removed, because the PECL file with role doc will not be copied to build dir.

https://github.com/apache/skywalking-php/blob/737b4f65824bb4939faa3e9452b83c73bf5ddfe8/config.m4#L66-L68

**Comment by heyanlong** [comment]
Why needs LICENSE for build?

**Comment by jmjoy** [comment]
> Why needs LICENSE for build?

Actually, it is unnecessary.

File: package.tpl.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@ limitations under the License.
 	</notes>
 	<contents>
 		<dir name="/">
-			{% for file in files %}<file role="src" name="{{ file.path }}" />
```
Comment (jmjoy) [inline package.tpl.xml:58]: Why not just make role as an placeholder, instead of using filter?

Comment (heyanlong) [reply package.tpl.xml:58]: Use filters to improve code judgment.

**Comment by heyanlong** [comment]
@jmjoy Could you review it?

**Pull request merged**